### ISSUE TITLE
Design Intent - Accessibility Feedback - GI Bill Comparison Tool, Institution Ratings System

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#50871]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions
Since you're a new team, I wanted to share a few accessibility resources that will be good to hold onto:
- Two super useful channels in DSVA Slack, [#inclusive-research](https://dsva.slack.com/archives/C033ZS9E7EW) and [#accessibility-help](https://dsva.slack.com/archives/C8E985R32). 
- Angela Fowler hosts accessibility office hours twice a week (announcements posted in #accessibility-help). Angela is a full-time screen reader user and will be able to give you some really good feedback outside of/in addition to what you get during usability testing.
- One last plug: If you ever need additional accessibility help, the Shared Support team can also jump in. Posting in #accessibility-help is a good way to get their attention!

## Feedback
- Practice areas will document their feedback following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback).

**Must:**
- As you move from medium-fidelity to high-fidelity, make sure the color combinations have sufficient contrast. For the progress indicators, make sure you have a minimum contrast ratio of 3:1 (and higher than that is going to be better) for the filled-in versus not-filled-in parts of the progress bar.

**Should:**
- Now is a good time to start thinking about what kind of text alternatives are needed for the star ratings and the progress indicators (and any other visuals you end up including). Best way to do that: Read your wireframe to someone else, without them being able to see it. Whatever you say for the visuals that isn't already text on the page should be included as a text alternative. If there are visuals that you skip over because there's duplicative text on the page already, then those visuals can be hidden from screen readers.
- As a best practice, annotate your designs with your decisions about text alternatives or other accessibility features (or even create a "screen reader transcript" artifact for what it should sound like) to share at your Midpoint Review. That makes any feedback I share at Midpoint a lot easier for both of us!
- In the meeting, Shane recommended testing with screen reader users and screen magnification users when you do usability research. I'll echo that here. Note that both of those groups are often unable to test with Sketch-generated wireframes, so you may need to explore alternative approaches for your research. (The #inclusive-research channel will be helpful here!)

**Consider:**

## Platform directions
- Remove the governance-team label
- Update "Issue Title"
- Link to collab cycle Request epic
- Add your feedback
- Remove the CC-Request label (only if you created the ticket using the New Issue link from the Collaboration Cycle Reviews board)
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)
